tests/kernel/install_ltp.pm

```diff
@@ -314,9 +314,12 @@ sub run {
 
     $self->select_serial_terminal;
 
+    # jsc#SLE-9743
+    $grub_param = 'debug_pagealloc';
```

Comment:
Reading the kernel patch a bit more closely; it is actually `debug_pagealloc=on`.

Reply:
Fixed.